src/control/lib/daos/container_property.go

```diff
@@ -549,7 +553,7 @@ var propHdlrs = propHdlrMap{
 	// ----------------------------------------
 	C.DAOS_PROP_ENTRY_LAYOUT_TYPE: {
 		C.DAOS_PROP_CO_LAYOUT_TYPE,
-		"Layout Type",
+		"Layout Type (POSIX, HDF5, PYTHON, SPARK, DATABASE, ROOT, SEISMIC, METEO)",
```

Comment:
I wonder if it might be helpful to have a slice defined somewhere, so it's easy to list all possible values.